app/views/jobs/index.html.erb

```diff
@@ -1,5 +1,58 @@
-All Jobs for <strong><%= @company.name %></strong>
+<% if @location %>
+    <h1 class="index-title">All Jobs in <strong><%= @location %></strong></h1>
+    <div id="links" class="row">
+    <div class="col-md-2" id="single-link">
+    <h5><%= link_to "Sort Jobs by Interest Level", "/companies/#{@company.id}/jobs?location=#{@location}&sort=interest" %></h5></div></div></div>
+      
+      <%= render partial: "job" %>
 
-<% @jobs.each do |job| %>
-  <ul><%= job.title %> at <%= job.company.name %></ul>
-<% end %>
+<% else %>
+
+  <h1 class="index-title">All Jobs at <strong><%= @company.name %></strong></h1>
+  <div id="links" class="row">
+    <div id="link" class="col-md-2">
+      <h5><%= link_to "Sort Jobs by Location", "/companies/#{@company.id}/jobs?sort=location" %></h5>
+    </div>
+    <div id="link" class="col-md-2">
+      <h5><%= link_to "Sort Jobs by Interest Level", "/companies/#{@company.id}/jobs?sort=interest" %></h5>
+    </div>
+    <div id="link" class="col-md-2">
+      <h5><%= link_to "Create a New Job at #{@company.name}", new_company_job_path(@company) %></h5>
+    </div>
+  </div>
+  </br></br>
+
+  <% if @contacts.count > 0 %>
```

Comment:
this is too much logic in your view - you could change it to `@contacts.any?` or creating a custom method?